### PR TITLE
Refactor `ContactPatientUiRenderer`

### PR DESCRIPTION
**Pull request by msasikanth**

### CODE REVIEW

File: app/src/main/java/org/simple/clinic/contactpatient/ContactPatientBottomSheet.kt [hunk 1]
```diff
@@ -316,11 +316,11 @@ class ContactPatientBottomSheet : BaseBottomSheet<
     )
```
Comment (Honey14) [inline app/src/main/java/org/simple/clinic/contactpatient/ContactPatientBottomSheet.kt:316]: Multiple things are happening in one commit, can we rename and remove in two different commits

**Review by Honey14 — CHANGES_REQUESTED** [review]
LGTM, left feedback. PTAL

File: app/src/main/java/org/simple/clinic/contactpatient/ContactPatientBottomSheet.kt (hunk 1 above)
Comment (msasikanth) [reply app/src/main/java/org/simple/clinic/contactpatient/ContactPatientBottomSheet.kt:316]: It's mostly just reshuffling of functions, extracting them out. I can separate the renaming functions to a new commit, though.

Comment (msasikanth) [reply app/src/main/java/org/simple/clinic/contactpatient/ContactPatientBottomSheet.kt:316]: done ✅

**Review by Honey14 — APPROVED** [review]
LGTM

**Pull request merged**